core/workflow-operator/build.sbt

```diff
@@ -95,7 +95,6 @@ libraryDependencies ++= Seq(
   "org.apache.commons" % "commons-compress" % "1.23.0",
   "org.tukaani" % "xz" % "1.9",
   "com.univocity" % "univocity-parsers" % "2.9.1",
-  "edu.stanford.nlp" % "stanford-corenlp" % "4.5.4",
-  "edu.stanford.nlp" % "stanford-corenlp" % "4.5.4" classifier "models",
+  "org.apache.lucene" % "lucene-analyzers-common" % "8.11.4",
```

Comment:
is this intended?

Reply:
Yes. It's needed inorder for Dictionary Matcher Op to use `org.apache.lucene.analysis.en.EnglishAnalyzer`. The packaged was included as part of Stanford CoreNLP, since it's removed we need to include `lucene-analyzers-common` manually.